hyrax.gemspec

```diff
@@ -67,7 +67,7 @@ EOF
   spec.add_dependency 'active_attr', '~> 0.9.0'
   spec.add_dependency 'redlock', '>= 0.1.2'
   spec.add_dependency 'retriable', '>= 2.9', '< 4.0'
-  spec.add_dependency 'active-fedora', '>= 11.1.3'
+  spec.add_dependency 'active-fedora', '~> 11.2'
```

Comment:
The intention was to have it in hyku, at least.  I don't understand the stack entirely and what all the community use cases are.  I assumed the update belongs on hyrax and then it gets pulled into hyku.  Also, this PR at least prompts consideration for this update in hyrax now that 11.2 is released.  I assumed an 11.x release is backward compatible and only releases new features and bug fixes.